src/Edm/Validation/Internal/InterfaceValidator/VisitorOfIPropertyValue.php

```diff
@@ -10,10 +10,10 @@
 
 class VisitorOfIPropertyValue extends VisitorOfT
 {
-    protected function VisitT($value, array &$followup, array &$references): iterable
+    protected function VisitT($value, array &$followup, array &$references): ?iterable
     {
         assert($value instanceof IPropertyValue);
-        return $value->getName() == null ?
+        return null === $value->getName() ?
```

Comment:
@c-harris , same nullability confusion as in VisitorOfIBinaryType - IPropertyValue->getName() is treated as nullable here, but is treated as non-nullable in EdmStructuredValue->computePropertiesDictionary.